pages/audit/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: 'guide'
+title: 'What is a Tracking Plan audit?'
+#abstract: 'Learn how to use Avo to identify issues in your existing tracking plan'
```

Comment:
Should we remove comments like this?

Reply:
I think it's fine to keep it in there as a reminder that a subtitle is missing here, and this is how you could add one. Does that make sense?